src/components/chip/chip.module.scss

```
@@ -216,18 +234,19 @@
   border-radius: 50%;
 
   &:hover:not(:disabled) {
-    background: rgba(0, 0, 0, 0.08);
+    background: color-mix(in srgb, currentColor 8%, transparent 92%);
```

Stylelint (17.5.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)